File: isips/castle_game/Assets/Script/camera/Panneau.js
#pragma strict

var canvas : Canvas;

function Start () {
	canvas.enabled = false;
}

function OnTriggerEnter2D (other : Collider2D) {
	if(other.gameObject == "chevalier"){
		canvas.enabled = true;
	}
}

function OnTriggerExit2D (other : Collider2D) {
	if(other.gameObject == "chevalier"){
		canvas.enabled = false;
	}
}

File: isips/castle_game/unity/castle_game/Assets/Script/camera/Panneau.cs
using UnityEngine;
using System.Collections;

public class Panneau : MonoBehaviour {
	public Canvas canvas;


	// Use this for initialization
	void Start () {
		canvas.enabled = false;
	
	}
	
	// Update is called once per frame
	void Update () {

	}
	void OnTriggerEnter2D(Collider2D other)
			{
			
		if(other.name == "Chevalier de la RoseCulotte")
					{
						canvas.enabled = true;
					}
			}
	void OnTriggerExit2D(Collider2D other) 
			{
			
		if(other.name == "Chevalier de la RoseCulotte")
					{
						canvas.enabled = false;
					}
			}
}